charts/dave/values-openshift.yaml

```
@@ -0,0 +1,32 @@
+# Dave EAI
+eai:
+  securityContext: null
+
+# Dave Document-Storage
+document-storage:
+  securityContext: null
+
+# Dave Geodata-EAI
+geodata-eai:
+  securityContext: null
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)